api/envoy/config/core/v3/http_uri.proto

```diff
@@ -28,7 +28,8 @@ message HttpUri {
   //
   //    uri: https://www.googleapis.com/oauth2/v1/certs
   //
-  string uri = 1 [(validate.rules).string = {min_len: 1}];
+  string uri = 1
+      [(validate.rules).string = {min_len: 1 well_known_regex: HTTP_HEADER_NAME strict: false}];
```

Comment:
This is not the right constraint. URI can contain characters that are not allowed in the header name. For example `/` and `:`

Reply:
oops, thanks for catching that!

It turns out that PGV has not implemented the URI check yet : `UnimplementedException: C++ URI validation is not implemented`